report/styles.md

```diff
@@ -0,0 +1,26 @@
+---
+title: soTired - Report
+author:
+  - Carina Strehler
+  - Anna Gsell
+  - Kilian Drechsler
+  - Lukas Diehm
+  - Marcel Arndt
+date: \today{}
+lang: en-GB
+documentclass: scrartcl
+mainfont: Arial
```

Comment:
I am a huge fan of serif fonts in reports, so Arial is sans-serif? Maybe we can discuss this tomorrow after our daily meeting?